specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-04-01-preview/examples/ApiManagementDeletePolicyFragment.json

```diff
@@ -0,0 +1,14 @@
+{
+  "parameters": {
+    "serviceName": "apimService1",
+    "resourceGroupName": "rg1",
+    "api-version": "2021-04-01-preview",
+    "subscriptionId": "subid",
+    "policyFragmentId": "policyFragment1",
+    "If-Match": "*"
+  },
+  "responses": {
+    "200": {},
```

Comment:
Need actual responses added

Reply:
Also - why are we returning 200 for a delete operation?  That shouldn't happen - it should be 204 only (if successful)

Reply:
Doesn't the RPC say 200 if it was deleted just now, 204 if it was already deleted?

Reply:
> Need actual responses added

What do you mean? It's a delete operation so we just return the status code. None of the other delete examples in APIM return anything else either.

Reply:
> Doesn't the RPC say 200 if it was deleted just now, 204 if it was already deleted?

Yes. All other delete endpoints do this as well.